packages/runtime/datastore/src/remoteChannelContext.ts

```diff
@@ -94,6 +94,26 @@ export class RemoteChannelContext implements IChannelContext {
 				this.id,
 			);
 
+			//* TODO: Read from snapshot instead (like in DataStoreRuntime)
```

Comment:
Highlighting this - Strategy 2 is not fully implemented. (And I don't think we have test coverage of DDS attach scenario anyway yet)